lib/transformers/transformers.js

```diff
@@ -125,8 +125,10 @@ const mapInRes = (res, req, startedAt, reqId, opts) => {
   let userAgent = getUserAgent(res.headers);
 
   const headers = extractHeaders(opts.headersRegex, opts.headerValueCallback, res.headers);
+  const hostFieldName = opts.hostFieldName || 'host';
```

Comment:
This fallback is added for safety reasons, as tests initialize a new adapter and pass options on the fly